### DISCUSSION TITLE
希望大佬能搞个singbox版本的

### DISCUSSION THREAD

**Discussion by hackerlengyue** [kkkgo/PaoPaoGateWay#82] [想法]
singbox感觉目前会更好用的多，https://www.youtube.com/watch?v=Jz6qBYTTfuE
这个视频里面的就会超级方便不少，更新订阅规则会方便很多。要是大佬做singbox版本的感觉会比目前clash的更好用一些。
只是单纯想法哈，别喷我

**Comment by kkkgo** [comment]
没看出有啥优势，你想要订阅就用suburl模式。
